.github/workflows/CI.yml

```diff
@@ -55,6 +55,12 @@ jobs:
       - run: cargo run --release -- --no-manual-traits ../glib-macros
         working-directory: checker
         if: matrix.rust == 'nightly' && matrix.conf.name == 'glib'
+      - name: Check doc aliases
+        run: |
+          python3 doc_aliases.py ../${{matrix.conf.name}}
+          cd .. && git diff -R --exit-code
```

Comment:
Just like what I wrote in the gstreamer-rs MR, it would make more sense to not use `-R` here :)

Reply:
Ok!

Reply:
`cargo fmt check` does the same btw